README.md

```diff
@@ -15,12 +15,20 @@ To ensure the long-term health of the project, we are temporarily pausing all of
 
 ✅ We will continue reviewing and merging community PRs.
 
+✅ No CI Changes at all - Linting, Testing, e2e testing are all happening on every single pull request.
```

Comment:
Does that mean you'll continue managing CI?

Did you mean?
"All our code will keep the same guarantee levels: Linting, Testing (including E2E tests) will continue to be maintained and applied on every PR. Fixing and improving CI is an important part of this phase. Main branch's CI will continue to publish artifacts for the purpose of testing."